src/main/java/carracing/vo/Car.java

```diff
@@ -0,0 +1,47 @@
+package carracing.vo;
+
+import carracing.util.RandomUtil;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Car {
+
+    private final String RACING_RESULT_MARK = "-";
+
+    private int carCount;
+    private int tryCount;
+
+    private List<String> racingResult;
+
+    public void initializeRacingResult() {
+        this.racingResult = Stream.generate(String::new).limit(this.carCount).collect(Collectors.toList());
```

Comment:
```suggestion
        this.racingResult = Stream.generate(String::new)
            .limit(this.carCount)
            .collect(Collectors.toList());
```
이렇게 개행을 해주면 더 보기 좋을 것 같아요~